scripts/facts-tracker/index.js

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+var fs = require('fs');
+var execSync = require('child_process').execSync;
+
+function escape(value) {
+  return '\'' + value.replace(/'/g, "'\\''") + '\'';
+}
+
+function exec(command) {
+  console.log('>', command.replace(process.env.GITHUB_TOKEN, '************'));
```

Comment:
It's really really useful to log all the commands the script is doing in order to troubleshot what's going on, especially since we can't ssh into a travis box and debug.

Do you think we shouldn't log commands?